### PR TITLE
PE-d bug fix round 2

### LINKED ISSUE #258: [PE-D][Tester C] Unclear demonstration in UG
The UG provides screenshots on how to use the find command.

However, the screenshots only show the result of running the command, and it is unclear what parameters were used to obtain this results, for instance what parameters were used for "s/" in the "find s/" command.

-------------
Labels: `type.DocumentationBug` `severity.VeryLow`
original: hj235/ped#5

### LINKED ISSUE #266: [PE-D][Tester E] Missing dependency in UI class diagram
The PersonPane component should have a dependency arrow pointing towards the Model component since it depends on the Person class to display the selected person details when using `show` command.

This is similar to how the PersonCard component depends on the Model component to display the person's details.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: yong-zhuo/ped#8

### LINKED ISSUE #281: [PE-D][Tester A] Pictures in help user guide
The user guide for the help commands shows multiple screenshot of the help window. It makes the user guide feels cluttered and the formatting to have a large white space.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: DesSnowy/ped#10

### LINKED ISSUE #299: [PE-D][Tester C] Overlap of ui components for persons with many listings
A person with many listings could have its tags overlap with other details on the show view. This could obstruct important information and make the product unusable under certain circumstances where a person has many listings, as the full details of the person could be unobtainable by the user.

To recreate: Add many tags to a single person, make the window as small as possible.

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: hj235/ped#10

### PR DESCRIPTION
**Pull request by tayxuenye**
Resolves #299 
Resolves #281 
Resolves #266 
Resolves #258

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -116,7 +116,9 @@ First, we need to make sure your computer has [Java](#technical-terms) 17 instal
 
 ### Step 3: Try Your First Commands
 
-Now that PROperty is running, let's try using the different commands available. All you have to do is to type the command in the command box and press Enter to execute it. 
+Now that PROperty is running, let's try using the different commands available. 
+We recommend you use a window size of at least 1920 x 1080 pixels to ensure all elements are displayed correctly.
+All you have to do is to type the command in the command box and press Enter to execute it.
```
Comment (sayomaki) [inline docs/UserGuide.md:121]: Maybe can improve the spacing on this one, or even convert the window recommendation into a banner (the one with the background colour + icon like a tip) to improve the design.

**Review by sayomaki — COMMENTED** [review]
Some quick comments for the user guide, but otherwise everything else looks okay!

**Review by BuffWuff1712 — APPROVED** [review]
Apart from what @sayomaki said, everything else lgtm, will approve this first

File: docs/UserGuide.md (hunk 1 above)
Comment (tayxuenye) [reply docs/UserGuide.md:120]: Okie I changed it! Let me know it it's okay

File: docs/UserGuide.md [hunk 2]
```diff
@@ -127,6 +128,9 @@ Some example commands you can try are:
 
 * `listing add 2 t/HDB a/Adam street, block 456, #03-03` : Adds a listing for the client with index 2 in PROperty.
 
+<div markdown="block" class="alert alert-primary">
+:bulb: Recommended Window Size<br> For the best experience, use a window size of at least 1920 x 1080 pixels.
```
Comment (Vatinius) [inline docs/UserGuide.md:132]: Might face some criticism by users who claim they do not know how to set their window to this specific dimensions.

Would it be better if we just tell the user to "full screen" the application?

Comment (tayxuenye) [reply docs/UserGuide.md:132]: Maybe can say "or full screen on your laptop"?

Comment (Vatinius) [reply docs/UserGuide.md:132]: But if we leave the pixel dimensions in there, a user might not know how to get to that size.

Comment (tayxuenye) [reply docs/UserGuide.md]: I see what you mean ok I changed it

**Pull request merged**